l10n_it_fatturapa_out/wizard/wizard_export_fatturapa.py

```diff
@@ -634,51 +649,52 @@ def setDettaglioLinea(
             # and '&#10;' would not be correctly visualized anyway
             # (for example firefox replaces '&#10;' with space
             Descrizione=encode_for_export(line.name.replace('\n', ' '), 1000),
-            PrezzoUnitario=('%.' + str(
-                price_precision
-            ) + 'f') % prezzo_unitario,
-            Quantita=('%.' + str(
-                uom_precision
-            ) + 'f') % line.quantity,
-            UnitaMisura=line.uos_id and (
-                unidecode(line.uos_id.name)) or None,
-            PrezzoTotale='%.2f' % line.price_subtotal,
+            PrezzoUnitario=('%.' + str(price_precision) + 'f') % float_round(
+                prezzo_unitario, price_precision),
+            Quantita=('%.' + str(uom_precision) + 'f') % line.quantity,
+            UnitaMisura=line.uos_id and (unidecode(line.uos_id.name)) or None,
+            PrezzoTotale='%.2f' % float_round(line.price_subtotal, price_precision),
```

Comment:
```suggestion
            PrezzoTotale='%.2f' % float_round(line.price_subtotal, 2),
```
il PrezzoTotale ha sempre una precisione pari a 2